src/containers/SellerProducts.jsx

```diff
@@ -0,0 +1,253 @@
+/* eslint-disable*/
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Please remove all comments pertaining to ESLint. It is important not to disable any ESLint rules. Instead, I encourage you to find ways to address the warnings and errors raised by ESLint in order to ensure code quality and adherence to best practices

Reply:
Done
I will keep that in mind in my next PR